backend/Cargo.toml

```diff
@@ -6,11 +6,15 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+common = { path = "../common" }
+dotenv = "0.15.0"
+
 rocket = { version = "=0.5.0-rc.3", features = ["json"] }
+rocket_db_pools = { version = "=0.1.0-rc.3", features = ["sqlx_postgres"]}
+
 serde = { version = "1.0.130", features = ["derive"] }
 serde_json = "1.0.68"
-sqlx = { version = "0.6.3", features = ["postgres", "runtime-tokio-rustls", "migrate", "uuid"]}
-rocket_db_pools = { version = "=0.1.0-rc.3", features = ["sqlx_postgres"]}
-dotenv = "0.15.0"
```

Comment:
I think we should use dotenvy because its the same implementation but it's got active devs and some additional features I havent explored yet, dotenv was abandoned but they're gonna merge back at some point

Reply:
I'm not adding `dotenv` here, just reordred it in the file.  I'm not against a better maintained version